Lib/fontTools/feaLib/lexer.py

```diff
@@ -45,13 +47,13 @@ def __init__(self, text, filename):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments=False):  # Python 2
```

Comment:
> No mention of extra parameters

sorry, I miss-typed `cannot not`, I meant `cannot`: no extra parameters are allowed in the iterator API.